task/build_cli.js

```diff
@@ -29,18 +31,19 @@ const builtStyle = build(style, options);
 outPath = path.join(buildDir, 'built-style.json');
 fs.writeFileSync(outPath, builtStyle, 'utf8');
 
-options.webfont = false; /* we don't need build sprite & font twice */
-options.needSprite = false
+// options.webfont = false; // we don't need build sprite & font twice */
+// options.needSprite = false
+// options.pins = false
 
-options.output = 'debug'
-const builtStyleDebug = build(style, options);
-outPath = path.join(buildDir, 'built-style-debug.json');
-fs.writeFileSync(outPath, JSON.stringify(JSON.parse(builtStyleDebug), null, 2), 'utf8'); //trick to restore formatting
+// options.output = 'debug'
+// const builtStyleDebug = build(style, options);
+// outPath = path.join(buildDir, 'built-style-debug.json');
+// fs.writeFileSync(outPath, JSON.stringify(JSON.parse(builtStyleDebug), null, 2), 'utf8'); //trick to restore formatting
 
-options.output = 'omt';
-const builtStyleOmt = build(style, options);
-outPath = path.join(buildDir, 'style-omt.json');
-fs.writeFileSync(outPath, builtStyleOmt, 'utf8');
+// options.output = 'omt';
+// const builtStyleOmt = build(style, options);
+// outPath = path.join(buildDir, 'style-omt.json');
+// fs.writeFileSync(outPath, builtStyleOmt, 'utf8');
```

Comment:
To revert before merging, I suppose.

Reply:
Yep, I'll restore it when I'm sure it doesn't mess with the build process. Erdapfel uses another path though, so it should be ok. But it broke the CLI version -_-